plugins/mcreator-localization/help/fr_FR/biome/max_water_depth.md

```diff
@@ -0,0 +1,2 @@
+Ce paramètre contrôle le nombre de blocs que le tronc de l'arbre personalisé peut avoir dans l'eau.

```

Comment:
Use this `Ce paramètre contrôle de combien de blocs le tronc de l'arbre personnalisé peut être immergé.`